ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx

```diff
@@ -1,37 +1,23 @@
 import React from 'react';
 import { CodeEditor, CodeEditorControl, Language } from '@patternfly/react-code-editor';
-import {
-    DownloadIcon,
-    MoonIcon,
-    ProcessAutomationIcon,
-    SunIcon,
-    UndoIcon,
-} from '@patternfly/react-icons';
+import { DownloadIcon, MoonIcon, SunIcon } from '@patternfly/react-icons';
 
 import { useTheme } from 'Containers/ThemeProvider';
 import download from 'utils/download';
 
 type NetworkPoliciesYAMLProp = {
     yaml: string;
-    generateNetworkPolicies: () => void;
```

Comment:
I'm removing the ability to do the simulator actions in those little icons on top of the code editor and creating a separate component for those. It was looking a little messy code-wise with the addition of the upload icon and visually it might look better to do it the way the mocks originally intended. Alan said using the icons was fine, so that's why I was doing it like that before